clients/drcachesim/tracer/raw2trace.h

```diff
@@ -936,7 +1042,7 @@ class raw2trace_t {
         uint last_cpu_ = 0;
         app_pc last_pc_if_syscall_ = 0;
 
-        std::set<app_pc> encoding_emitted;
+        bitset_hash_table_t<app_pc> encoding_emitted;
```

Comment:
This is not in the PR description: please add it there.

What % of speedup you see is due to this vs lz4?

Reply:
bitset_hash_table_t have us about 20% speed up